composer.json

```diff
@@ -177,7 +177,8 @@
         "platform": {
             "php": "5.6.0"
         },
-        "sort-packages": true
+        "sort-packages": true,
+        "process-timeout": 0
```

Comment:
bad idea, we need to figure out why and since when our tests are so slow.

Reply:
300 seconds to run tests is very short.
Recently we added about 200 integration/e2e tests. Do you really think the 300 seconds timeout is enough ? ^^

Reply:
phpunit-legacy does really few unit tests ^^
they are almost all end to end tests, using database fixtures, several code layers, models and more.
In unicorn world we should be able do make strict unit tests here. We cannot.

Reply:
I'm sure we can do something for it, 2 performance issues:
* re-dump the database (we could make it more performant /c @jocel1 )
* and a new one that come from AbstractCartTest

https://blackfire.io/profiles/1f5903f5-42ca-4715-aff1-3e166aa2edc6/graph?settings%5Bdimension%5D=wt&settings%5Bdisplay%5D=landscape&settings%5BtabPane%5D=nodes&selected=Tests%5CTestCase%5CIntegrationTestCase%3A%3AsetUpBeforeClass&callname=main()

I repeat, I'm against timeout: 0 value: we need to setup a decent time, but not unlimited one.

Reply:
travis limit is 50 min
10min is the timeout without any log

ok for 50min ^^

Reply:
why I do not want any timeout on the composer task is because it is injecting coupling between tests and their execution time : false fail wil be thrown on timeout. It assumes a coupling between tests and the hardware they are running on (locally I didn't have any timeout issue since my local hardware seems faster than travis one)

timeout should be set in job config, not in test config ;)
Else we always face with scalability issues

Reply:
@mickaelandrieu if you prefer I can remove the line from composer and add the "--ignore-timeout" option on travis config file when running tests ;)

Reply:
Okay for adding `--ignore-timeout` for travis tests. We can look up why tests are slow and look for ways of improving it, but now is not that time.